airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/jinja.py

```diff
@@ -2,36 +2,51 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
+import ast
 import datetime
+import numbers
 
 from airbyte_cdk.sources.declarative.interpolation.interpolation import Interpolation
+from dateutil import parser
 from jinja2 import Environment
 from jinja2.exceptions import UndefinedError
 
 
 class JinjaInterpolation(Interpolation):
-    def __init__(self):
+    def __init__(self, datetime_format: str = "%Y-%m-%d"):
         self._environment = Environment()
         # Defines some utility methods that can be called from template strings
         # eg "{{ today_utc() }}
         self._environment.globals["now_local"] = datetime.datetime.now
         self._environment.globals["now_utc"] = lambda: datetime.datetime.now(datetime.timezone.utc)
         self._environment.globals["today_utc"] = lambda: datetime.datetime.now(datetime.timezone.utc).date()
+        self._environment.globals["timestamp"] = (
+            lambda dt: int(dt)
+            if isinstance(dt, numbers.Number)
```

Comment:
What about something like RFC3339? is that intentionally not supported?

Reply:
I think it's better to give users exactly a much more limited subset of available formats e.g: `YYYY-MM-DD` or `YYYY-MM-DDT00:00:00`.

Reply:
added a test with rfc3339 to confirm it's supported https://github.com/airbytehq/airbyte/pull/14268/files#diff-191886aa5db38483b56930efef8f6bee7dcec1854fa1fb6a248b3f9b5839ee4bR40